### PR TITLE
Client - Misalignment between inline language selector and brand Navbar items #145

### PR DESCRIPTION
**Pull request by MuhweziDeo**
Fix small misalignment when language selector is in inline mode

### CODE REVIEW

File: geonode_mapstore_client/client/themes/geonode/scss/_action-navbar.scss [hunk 1]
```diff
@@ -114,6 +114,9 @@
             border-radius: 0.25rem;
         }
         .gn-language-selector {
+            .inline {
```
Comment (allyoucanmap) [inline geonode_mapstore_client/client/themes/geonode/scss/_action-navbar.scss:117]: this selector is not valid for the class name you composed before

File: geonode_mapstore_client/client/themes/geonode/scss/_action-navbar.scss [hunk 2]
```diff
@@ -114,6 +114,9 @@
...
+                padding-right: 2.8rem;
```
Comment (allyoucanmap) [inline geonode_mapstore_client/client/themes/geonode/scss/_action-navbar.scss:118]: If I try to test this value in the dev tool this is the resoult
image

The value should be 0.5rem instead

**Review by allyoucanmap — CHANGES_REQUESTED** [review]
The inline option can be added in static/mapstore/configs/localConfig.json at the path `geoNodeConfiguration.theme.languageSelector.inline` for testing pourpose

```js
"geoNodeConfiguration": {
    "theme": {
        // ...,
        "languageSelector": {
            "inline": true
        }
    },
```

**Review by allyoucanmap — CHANGES_REQUESTED** [review]

File: geonode_mapstore_client/client/js/components/home/LanguageSelector.jsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ const LanguageSelector = forwardRef(({
     return (
         <div
             ref={ref}
-            className="gn-language-selector"
+            className={`gn-language-selector ${inline} ? 'inline' : ''`}
```
[the remark this answers is not in the document]
Comment (MuhweziDeo) [reply geonode_mapstore_client/client/js/components/home/LanguageSelector.jsx:35]: I do not know how I didnot see this

**Review by allyoucanmap — APPROVED** [review]

**Pull request merged**